sympy/polys/matrices/tests/test_ddm.py

```diff
@@ -346,3 +346,28 @@ def test_DDM_charpoly():
 
     A = DDM([[ZZ(1), ZZ(2)]], (1, 2), ZZ)
     raises(DDMShapeError, lambda: A.charpoly())
+
+
+def test_DDM_getitem():
+    dm = DDM([
+        [ZZ(1), ZZ(2), ZZ(3)],
+        [ZZ(4), ZZ(5), ZZ(6)],
+        [ZZ(7), ZZ(8), ZZ(9)]], (3, 3), ZZ)
+
+    assert dm[1][1] == ZZ(5)
+
+    raises(IndexError, lambda: dm[3][3])
```

Comment:
negative valid and invalid indices should also be tested.

Reply:
Sure, done.
Thanks.